docs/source/notes/ddp.rst

```diff
@@ -65,7 +65,14 @@ updated, and all models on different processes should be exactly the same.
         os.environ["MASTER_PORT"] = "29500"
         main()
 
+DDP works with TorchDynamo.  When used with TorchDynamo, apply the DDP model wrapper
+before compiling the model, such that torchdynamo can apply ``DDPOptimizer``
+(graph-break optimizations) based on DDP bucket sizes.  (See section below for more detail TODO link)
 
+.. code::
+
+        ddp_model = DDP(model, device_ids=[rank])
+        ddp_model = torch.compile(ddp_model)
```

Comment:
So we can't merge this quite yet since won't exist until sometime next week, in the meantime you can can use the optimize API if you'd rather merge this now

Reply:
no rush, i can just land it once its ready. keep me posted.